Tests/MongoSwiftSyncTests/MongoCollection+IndexTests.swift

```diff
@@ -113,6 +120,7 @@ final class MongoCollection_IndexTests: MongoSwiftTestCase {
         // testOptions index
         var expectedTestOptions = options
         expectedTestOptions.name = "testOptions"
+        expectedTestOptions.version = 2
```

Comment:
so serverless won't let you specify an index version but returns one either way?

Reply:
Yeah that appears to be the case.